odoo/addons/base/models/ir_mail_server.py

```diff
@@ -204,20 +204,21 @@ def _get_test_email_addresses(self):
         self.ensure_one()
         email_to = "noreply@odoo.com"
         if self.from_filter:
-            if "@" in self.from_filter:
+            from_filter_parts = [part.strip() for part in self.from_filter.split(",")]
```

Comment:
Don't we have a tool for that? I see "email_split" in tools.

EDIT: Actually it looks like we can use domains so this does not work :(